tools/benchmark/binsize/src/main.rs

```diff
@@ -0,0 +1,26 @@
+// This file is part of ICU4X. For terms of use, please see the file
+// called LICENSE at the top level of the ICU4X source tree
+// (online at: https://github.com/unicode-org/icu4x/blob/main/LICENSE ).
+
+use std::fs;
+
+fn wasm_filesize(dir: &str) -> Result<u64, std::io::Error> {
+    let paths = fs::read_dir(dir).expect("Directory wasm/ not found!");
+    for path in paths {
+        let p = path.unwrap().path();
+        if let Some(suffix) = p.extension() {
+            if suffix == "wasm" {
+                println!(
```

Comment:
Suggestion: There is an implicit assumption that stdout should be piped to a file. Please document this assumption.

Reply:
Added comment prior to line.